bin/setup.sh

```diff
@@ -38,5 +37,9 @@ cd /vagrant/
 git submodule init
 git submodule update
 
+
+#add dataset
+python manage.py makemigrations gis_csdt
```

Comment:
All migrations should come with / be premade. Make sure to commit your migrations.